students/khtruong/lesson_03/mailroom.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+
+"""This module runs the mailroom script.
+"""
+
+data = [['Alpha', 3.65, 54.50],
+        ['Beta', 36.54, 1.25, 54.87],
+        ['Gamma', 17.38],
+        ['Delta', 25.55, 33.33, 78.14],
+        ['Epsilon', 65.40]]
+
+
+def main():
+    """Return 3 options for user to select to run mailroom script."""
+    while True:
+        userinput = input('\nEnter "0" to "quit" \n'
+                          'Enter "1" to "Send a Thank You" \n'
+                          'Enter "2" to "Create a Report" \n'
+                          'What do you want to do? > ')
+        if int(userinput) == 0:
+            break
+        elif int(userinput) == 1:
```

Comment:
You could just do `elif userinput == '1':` since I don't think you need these to be converted to ints